siddhi-extensions/geo/src/main/java/org/wso2/siddhi/gpl/extensions/geo/stream/GeoCrossesStreamProcessor.java

```diff
@@ -52,7 +52,7 @@
      * @return the additional output attributes introduced by the function
      */
     @Override
-    protected List<Attribute> init(AbstractDefinition inputDefinition, ExpressionExecutor[] attributeExpressionExecutors, ExecutionPlanContext executionPlanContext, boolean outputExpectsExpiredEvents) {
+    protected List<Attribute> init(AbstractDefinition inputDefinition, ExpressionExecutor[] attributeExpressionExecutors, ExecutionPlanContext executionPlanContext) {
         geoOperation = new WithinOperation();
         geoOperation.init(attributeExpressionExecutors, 1, attributeExpressionLength);
         ArrayList<Attribute> attributeList = new ArrayList<Attribute>(1);
```

Comment:
Thanks for pointing it out @grainier. fixed with https://github.com/wso2-gpl/siddhi-extensions/pull/18/commits/5eee65245ae9297c84cca2364535fcb96ab8ddd6